src/components/InternalOverlay/index.js

```diff
@@ -134,13 +138,18 @@ InternalOverlay.propTypes = {
      * to an element after it is opened.
      */
     onOpened: PropTypes.func,
+    /**
+     * @ignore
+     */
+    children: PropTypes.node,
```

Comment:
To change the API of any component we have to have a discussion.

Reply:
Sorry for that @yvmunayev, we already discuss it, this is necessary since the other way with render prop we have issues since when use:
```
render={() => <div />}
```
this cause the component passed is new each time